unity-renderer/Assets/Scripts/MainScripts/DCL/Components/BuilderInWorld/DCLName.cs

```diff
@@ -17,6 +17,8 @@ public class DCLName : BaseDisposable
     public class Model : BaseModel
     {
         public string value;
+        //Note: This value is used for builder to manage the smart items, when the builder is no longer active we should remove it
```

Comment:
Only as suggestion: I would make this note as a `TO DO` in order to have it more localized in the future.

Reply:
Will do in the current branch that I am working!